src/test/scala/org/camunda/feel/interpreter/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -39,6 +39,17 @@ class BuiltinNumberFunctionsTest
     eval(" decimal(2.5, 0) ") should be(ValNumber(2))
   }
 
+  "A decimal() function with rounding mode" should "return number with a given scale" in {
+
+    eval(""" decimal(1.5, 0, "HALF_EVEN") """) should be(ValNumber(2))
+    eval(""" decimal(1.5, 0, "HALF_UP") """) should be(ValNumber(2))
+    eval(""" decimal(1.5, 0, "UP") """) should be(ValNumber(2))
+    eval(""" decimal(1.5, 0, "HALF_DOWN") """) should be(ValNumber(1))
+    eval(""" decimal(1.5, 0, "DOWN") """) should be(ValNumber(1))
+    eval(""" decimal(1.5, 0, "FLOOR") """) should be(ValNumber(1))
+    eval(""" decimal(1.5, 0, "CEILING") """) should be(ValNumber(2))
```

Comment:
We should provide more examples to see the differences in the rounding modes.

Additionally, we should add a test case if the given mode is not valid.